app/models/organization.rb

```diff
@@ -89,4 +89,16 @@ def scale_values
         k_size6:    items.find_by(name: "Kids (Size 6)").id,
       }
   end
+
+  private_class_method
+
+  def self.seed_items(org)
```

Comment:
Need to add a call to this. Not a fan of the location of a class method.

Reply:
Done. Added to `admins_controller`